### PR TITLE
Organize documentation a bit

### PR DESCRIPTION
**Pull request by jayantk**
I'm trying to organize the documentation a bit so that it has a reasonable flow. I'm following the suggestions of https://diataxis.fr/ by making the `documentation/` portion into "explanation", and then moving specific integration recipes into a separate top-level "how-to guides" section. We already have an API Reference that covers the "reference" portion, so all we're missing is "tutorials".

### CODE REVIEW

File: pages/documentation/pythnet-price-feeds/price-pusher.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Price Pusher
```
Comment (jayantk) [inline pages/documentation/pythnet-price-feeds/price-pusher.mdx:1]: Moved out of EVM docs.

note: this will need to be renamed to scheduler, but let's do that in a separate PR because it's going to touch a lot of things.

File: pages/documentation/tradingview-integration.mdx [hunk 2]
```diff
@@ -1,112 +0,0 @@
-import { Tab, Tabs } from "nextra-theme-docs";
```
Comment (jayantk) [inline pages/documentation/tradingview-integration.mdx:1]: moved to how-to guides

File: pages/how-to/how-to-create-tradingview-charts.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import { Tab, Tabs } from "nextra-theme-docs";
```
Comment (jayantk) [inline pages/how-to/how-to-create-tradingview-charts.mdx:1]: moved from documentation to the how-to section.

Comment (jayantk) [reply pages/how-to/how-to-create-tradingview-charts.mdx:1]: I think we should update the "benchmarks" section of the docs to talk about how the API supports charting via tradingview, then link users here for how to do it.

**Review by guibescos — APPROVED** [review]

**Review by KemarTiti — APPROVED** [review]
lgtm
